Backpack/Panel/Classes/BPKPanel.h

```diff
@@ -32,6 +40,12 @@ NS_SWIFT_NAME(Panel) IB_DESIGNABLE @interface BPKPanel : UIView
  */
 @property(nonatomic, assign, getter=isPadded) IBInspectable BOOL padded;
 
+/**
+ * The style of the panel to use.
+ * @see BPKButtonStyle
+ */
+@property(nonatomic, assign) BPKPanelStyle style;
```

Comment:
We should replace this `assign` for a `weak` or `strong` attribute instead. 
"_Assign is the older version of weak attribute_"